services/network_acl_manager/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+#server.port=9000
```

Comment:
server.port=9008 

This is the default port for NACL manager. The default port for other services could be referred to https://github.com/futurewei-cloud/alcor/blob/master/services/port_manager/src/main/resources/application.properties (updated recently)

This will help to avoid port conflict for e2e integration.